### Hadi Rahmani

<!--
**hrahmanin/hrahmanin** is a âœ¨ _special_ âœ¨ repository because its `README.md` (this file) appears on your GitHub profile.

Here are some ideas to get you started:

- ğŸ”­ Iâ€™m currently working on ...
- ğŸŒ± Iâ€™m currently learning ...
- ğŸ‘¯ Iâ€™m looking to collaborate on ...
- ğŸ¤” Iâ€™m looking for help with ...
- ğŸ’¬ Ask me about ...
- ğŸ“« How to reach me: ...
- ğŸ˜„ Pronouns: ...
- âš¡ Fun fact: ...
-->

ğŸ”¬ Computational Biologist | Biophysicist | Genomics  
ğŸ“ Based in California | Open to industry and academic collaborations

ğŸ§  My interests:
- Chromatin modeling & genome folding
- NGS data analysis (ChIP-seq, Hi-C)
- Machine learning in biology

ğŸ›  My projects:
- ğŸ§¬ **Predicting 3D Genome Folding from Single-Molecule Footprinting** *(GitHub link coming soon)*  
  *Developed machine learning models to infer 3D chromatin architecture using single-molecule footprinting data (e.g., methylation footprints, occupancy patterns). Integrated sequence features and occupancy patterns to predict structural genome organization.*
- ğŸ” [dynamic_extrusion_boundaries](https://github.com/hrahmani/dynamic_extrusion_boundaries): Simulating DNA loop extrusion with dynamic barriers.
- ğŸ“Š [chromoscores](https://github.com/hrahmani/chromoscores): Quantitative analysis tools for Hi-C maps.

ğŸ“« Let's connect:
[LinkedIn](https://www.linkedin.com/in/hadi-rahmani-phd-724225211/) Â· [Google Scholar](https://scholar.google.com/citations?user=UUYEU4UAAAAJ&hl=en&oi=ao) Â· [CV](#)

