### Hadi Rahmaninejad

<!--
**hrahmanin/hrahmanin** is a âœ¨ _special_ âœ¨ repository because its `README.md` (this file) appears on your GitHub profile.-->

Computational Biology | Biophysic | Genomics | Soft Matter 


My interests:
- Modeling 3D Genome folding and its regulatory role
- Applications of machine learning models in biology
- Developing bioinformatic pipeline for next-generation sequencing (NGS) data
  
  
<!--## ðŸ”¬ Research Focus

- **Machine Learning for Genomics:** Developing deep learning models to predict Genome architecture from genomic & epigenomic data
- **3D Genome Organization:** Modeling DNA loop extrusion using polymer physics
- **Molecular Simulations:** Simulating chromatin dynamics in OpenMM to quantify accessory protein impacts on genome structure
- **Multi-omics Pipelines:** Building scalable workflows (Nextflow, Singularity) for Hiâ€‘C, ATACâ€‘seq & ChIPâ€‘seq data processing-->

### Key Projects
#### Predicting 3D Genome Folding from Single-Molecule Footprinting <!--*(GitHub link coming soon)* [Sitewise occupancy prediction](https://github.com/Fudenberg-Research-Group/OccupancyInputCTCF)--> [Sitewise occupancy prediction](https://github.com/hrahmanin/OccuFold)
Deep learning model to infer 3D chromatin structure using single-molecule data (e.g., methylation footprints). Integrated sequence and occupancy data to predict genome folding and validate with Hi-C.
![Workflow Cartoon](https://github.com/Fudenberg-Research-Group/OccupancyInputCTCF/blob/main/figures/workflowfigurenews.png)

### [fastaFRiP ](https://github.com/Fudenberg-Research-Group/fastaFRiP)
Reproducible pipeline for ChIP-seq analysis, including FRiP score computation using Bowtie2, samtools, and MACS2. Built for easy adoption and reproducibility.

### [dynamic_extrusion_boundaries](https://github.com/Fudenberg-Research-Group/dynamic_extrusion_boundaries)
Mechanistic model of CTCF/cohesin loop extrusion using dynamic barriers. Built in Python + OpenMM. Validated predictions with Hi-C and compartment scores.
![Workflow Cartoon](https://github.com/Fudenberg-Research-Group/dynamic_extrusion_boundaries/blob/main/output/cartoons/cropped_output_mult_seq_bar_combination_size.gif)

### [ChromoScore](https://github.com/hrahmanin/chromoscores)
Python package to quantify structural features (e.g., TADs, loops) from contact maps. Optimized for simulated Hi-C data. *(Repo: chromoscores)*
![Workflow Cartoon](https://github.com/hrahmanin/chromoscores/blob/main/docs/representations.png)

### ðŸ§· looplib (Open2C contributor)
Contributed to modeling functionality and maintenance of open-source codebase, particularly looplib, a package for 1D DNA loop extrusion.

### ðŸ’» Skills & Tools
**Computational Biology & Bioinformatics:** NGS data analysis, Genomics & Multiâ€‘Omics data (Hiâ€‘C, ATACâ€‘seq, ChIPâ€‘seq, RNAâ€‘seq), Tools: Cooltools, Biopython, Bioconductor 
**Modeling & Simulation:** Molecular dynamics (OpenMM, GROMACS, AMBER), Finite element modeling: FEniCS  
**Machine Learning & Data Analysis:** ML models: clustering, regression, neural networks, CNNs, NLP, Libraries: PyTorch, scikitâ€‘learn, XGBoost  
**Programming & Scripting:** Python (NumPy, SciPy, Pandas),R, Bash / Shell scripting  
**Workflow & Pipeline Development:** Workflow management(Nextflow, Snakemake), Version control (Git, GitHub), Cloud & HPC computing(AWS, USC HPC), Containerization(Singularity)



### ðŸ“š Selected Publications [Google Scholar](https://scholar.google.com/citations?user=UUYEU4UAAAAJ)

- Rahmaninejad H, et al. *Dynamic Barriers Modulate Cohesin Positioning and Genome Folding at Fixed Occupancy*, Genome Research (2025)  
- Rahmaninejad H, et al. *Stimuli-Responsive Polymer Brushes in Nanofluidic Channels*, ACS Appl. Mater. Interfaces (2023)
- Rahmaninejad H, et al. *Crowding within synaptic junctions influences the degradation of nucleotides by CD39 and CD73 ectonucleotidases*, Biophysical Journal (2022) 
- Rahmaninejad H, et al. *Co-localization and confinement of ecto-nucleotidases modulate extracellular adenosine nucleotide distributions*, PLoS computational biology (2020)

#### ðŸ“ˆ GitHub Stats

![Hadi's GitHub stats](https://github-readme-stats.vercel.app/api?username=hrahmanin&show_icons=true&count_private=true&hide=issues)

---

<!-- ## ðŸŽ¨ Project Illustrations (Optional)

You can add cartoons/diagrams for each project by placing `.png`, `.svg`, or `.gif` files in your repo and referencing them like this:

```markdown
![ChromoScore Diagram](images/chromoscore_diagram.png)-->

