### Hadi Rahmani

<!--
**hrahmanin/hrahmanin** is a ✨ _special_ ✨ repository because its `README.md` (this file) appears on your GitHub profile.

Here are some ideas to get you started:

- 🔭 I’m currently working on ...
- 🌱 I’m currently learning ...
- 👯 I’m looking to collaborate on ...
- 🤔 I’m looking for help with ...
- 💬 Ask me about ...
- 📫 How to reach me: ...
- 😄 Pronouns: ...
- ⚡ Fun fact: ...
-->

🔬 Computational Biologist | Biophysicist | Genomics  
📍 Based in California | Open to industry and academic collaborations

🧠 My interests:
- Chromatin modeling & genome folding
- NGS data analysis (ChIP-seq, Hi-C)
- Machine learning in biology

🛠 My projects:
- 🧬 **Predicting 3D Genome Folding from Single-Molecule Footprinting** *(GitHub link coming soon)*  
  *Developed machine learning models to infer 3D chromatin architecture using single-molecule footprinting data (e.g., methylation footprints, occupancy patterns). Integrated sequence features and occupancy patterns to predict structural genome organization.*
- 🔁 [dynamic_extrusion_boundaries](https://github.com/hrahmani/dynamic_extrusion_boundaries): Simulating DNA loop extrusion with dynamic barriers.
- 📊 [chromoscores](https://github.com/hrahmani/chromoscores): Quantitative analysis tools for Hi-C maps.

📫 Let's connect:
[LinkedIn](https://www.linkedin.com/in/hadi-rahmani-phd-724225211/) · [Google Scholar](https://scholar.google.com/citations?user=UUYEU4UAAAAJ&hl=en&oi=ao) · [CV](#)

